components/MentorshipsList/MentorshipInfo.js

```diff
@@ -0,0 +1,51 @@
+import { Stack, VStack, Text, StackDivider } from "@chakra-ui/layout";
+import { RequestStatusEnum, RelationshipStatusEnum } from "../../types/dbTablesEnums";
+import ChakraUIRenderer from "chakra-ui-markdown-renderer";
+import ReactMarkdown from "react-markdown";
+
+const markdownTheme = {
+  p: (props) => {
+    const { children } = props;
+    return <Text mb={2}>{children}</Text>;
+  },
+};
+
+const MentorshipInfo = ({ mentorship }) => {
+  const { message, mentorshipDate, status, longterm } = mentorship;
+  let dateLabel = "";
+  if (status === RelationshipStatusEnum.Started) {
+    dateLabel = "Started on";
+  } else if (status === RequestStatusEnum.Pending) {
+    dateLabel = "Requested on";
+  }
+  return (
+    <>
+      <Stack
+        direction={"column"}
+        textAlign={"left"}
+        justifyContent={"center"}
+        width='100%'
+      >
+      
+      <VStack
+          divider={<StackDivider borderColor='gray.200' />}
+          spacing={4}
+          align='stretch'
+      >
+        { (dateLabel && mentorshipDate) && 
+          <Text textAlign={"right"} mt={4}>{dateLabel}: {mentorshipDate}</Text>
```

Comment:
```suggestion
          <Text mt={4}>{dateLabel}: {mentorshipDate}</Text>
```

Reply:
I think left aligning the content in the card could look better, WDYT?
![Screenshot 2023-02-14 at 08 22 11](https://user-images.githubusercontent.com/45630868/218681837-8824c430-a52c-4205-a247-def819afcdf2.png)
![Screenshot 2023-02-14 at 08 22 02](https://user-images.githubusercontent.com/45630868/218681863-48b846f9-2861-462e-badd-53693c0ba503.png)

Reply:
We could also format the date using something like this: `new Intl.DateTimeFormat().format(new Date("2022-08-06T22:25:37.873Z"))`